lib/improver/database.py

```diff
@@ -36,11 +36,11 @@
 
 import itertools
 import pandas as pd
+from pandas import DataFrame
 import numpy as np
 import sqlite3
```

Comment:
The pandas/numpy imports should be after the rest if we are ordering by standard library first

Reply:
Happy for them to be reordered as you want.